library/std/src/io/stdio.rs

```diff
@@ -988,36 +909,26 @@ pub(crate) fn clone_io() -> (Option<Box<dyn LocalOutput>>, Option<Box<dyn LocalO
 /// thread, it will just fall back to the global stream.
 ///
 /// However, if the actual I/O causes an error, this function does panic.
-fn print_to<T>(
-    args: fmt::Arguments<'_>,
-    local_s: &'static LocalKey<RefCell<Option<Box<dyn LocalOutput>>>>,
-    global_s: fn() -> T,
-    label: &str,
-) where
+fn print_to<T>(args: fmt::Arguments<'_>, global_s: fn() -> T, label: &str)
+where
```

Comment:
```suggestion
fn print_to<T>(
  args: fmt::Arguments<'_>,
  global_s: fn() -> T,
  label: &str
) where
```
Is there specific guidance on where to nest related elements for readability?

Reply:
> Is there specific guidance on where to nest related elements for readability?

Yes, we simply do whatever `rustfmt` does. (Run by `./x.py fmt` in the `rust` repository.)

Your suggestion would not pass the automated style check. It's automatically checked that the formatting matches what `rustfmt` would do.

Reply:
Ok thanks!